sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/NotSupportedAssetFileTypeException.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion
+{
+    /// <summary>
+    /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
+    /// </summary>
+    public class NotSupportedAssetFileTypeException : Exception, ISerializable
+    {
+        /// <summary>
+        /// Creates an instance of the UnsupportedAssetFileTypeException
+        /// </summary>
+        public NotSupportedAssetFileTypeException()
+            : base($"The provided asset file type is unsupported by Azure Object Anchors for conversion.")
+        {
+        }
+
+        /// <summary>
+        /// Creates an instance of the UnsupportedAssetFileTypeException
+        /// </summary>
+        /// <param name="message">The message corresponding to the exception</param>
+        public NotSupportedAssetFileTypeException(string message)
+            : base(message)
+        {
+        }
+
+        /// <summary>
+        /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
+        /// </summary>
+        /// <param name="message">The message corresponding to the exception</param>
+        /// <param name="inner">The inner exception</param>
+        public NotSupportedAssetFileTypeException(string message, Exception inner)
```

Comment:
It looks like we always use the internal ctor in the code. Are these three other ctors for testing? Could we just make the other one public and remove these three?

Reply:
See section 7.4 of the FXDG

Reply:
@tg-msft thoughts here? this was based on your feedback

Reply:
Yes - leave what you have.